### PR TITLE
Update description for user email, phone, and preferences

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

Update the text to match design.

## Test Plan

Manual:

image

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/console/project-[project]/auth/user-[user]/updateEmail.svelte [hunk 1]
```diff
@@ -36,6 +36,7 @@
 <Form onSubmit={updateEmail}>
     <CardGrid>
         <Heading tag="h6" size="7">Email</Heading>
+        <p>Update user's email. An Email should be formatted as: <b>name@example.com.</b></p>
```
Comment (eldadfux) [inline src/routes/console/project-[project]/auth/user-[user]/updateEmail.svelte:39]:
```suggestion
        <p>Update user's email. An Email should be formatted as: <b>name@example.com</b>.</p>
```

Do we have any option to set the value as a code example?

**Review by eldadfux — CHANGES_REQUESTED** [review]
Let's drop the usage of the <b> tags. It will be very difficult for us to decide when and when not use them.

File: src/routes/console/project-[project]/auth/user-[user]/updateEmail.svelte (hunk 1 above)
Comment (stnguyen90) [reply src/routes/console/project-[project]/auth/user-[user]/updateEmail.svelte:39]: Do you mean just formatting it like inline code? It's possible with pink, but we don't use it anywhere in the Console.

**Comment by stnguyen90** [comment]
@eldadfux, what did you mean by: 

> Let's drop the usage of the tags

Did you mean the `tag="h6"` on the `<Heading>`?

**Comment by eldadfux** [comment]
@stnguyen90 Sorry formatting for in the way. I meant `<b>` tags.

**Review by eldadfux — APPROVED** [review]

**Pull request merged**